src/Symfony/Bridge/ProxyManager/Tests/LazyProxy/PhpDumper/Fixtures/proxy-implem.php

```diff
@@ -96,7 +96,7 @@ public function __set($name, $value)
 
         $targetObject = $this->valueHolder%s;
 
-        return $targetObject->$name = $value;
+        $targetObject->$name = $value; return $targetObject->$name;
```

Comment:
Just for my curiosity, why writing this in one line?

Reply:
This is the way the code is generated after https://github.com/Ocramius/ProxyManager/pull/646